### ISSUE TITLE
[Documentation] Improve Clarity on `torchvision.io.write_video` options

### ISSUE THREAD

**Issue by N00bcak** [pytorch/vision#8509]
### tl;dr `options` parameter under `torchvision.io.write_video` is inadequately documented, challenging for end-users to tweak videos to their liking. same goes for `audio_options`

In [`torchvision.io.write_video`](https://pytorch.org/vision/stable/generated/torchvision.io.write_video.html#torchvision.io.write_video), `options` and `audio_options` parameters are not sufficiently elaborated upon.

At first glance, this is okay as the function is ultimately just a wrapper around `PyAV`'s video recording capabilities. 

In practice, however, **it is nontrivial to find relevant documentation to finetune the output.**
This is especially so if the end-user, like me, is not familiar with `PyAV` or video recording in general.

[Here is an example](https://github.com/pytorch/rl/issues/2258) of the potential issues this may cause. 

### Suggest a potential alternative/fix

This is an issue we are working on as well in [TorchRL](https://github.com/pytorch/rl/tree/4e60774dedb6d5847c163c985b71928598123b46).

As suggested by @vmoens, it would be nice if a link could be provided to the [FFmpeg documentation](https://trac.ffmpeg.org/wiki), which is ultimately behind the `options` exposed by `PyAV`.

A potential extra step could then be to translate the wiki information into *specific code examples* that end-users can easily read and apply. This will probably benefit more people than if such information were only being documented in downstream libraries. [Here's what a "specific code example" could look like](https://github.com/pytorch/rl/blob/4e60774dedb6d5847c163c985b71928598123b46/knowledge_base/VIDEO_CUSTOMISATION.md),

**Comment by NicolasHug** [comment]
Thanks for the report @N00bcak , I agree this is insufficiently documented. I'm happy to review a PR to improve the docs.
